build-jenkins-agent-windows.pkr.hcl

```diff
@@ -0,0 +1,76 @@
+build {
+  source "amazon-ebs.base" {
+    name           = "windows"
+    communicator   = "winrm"
+    user_data_file = "./provisioning/setupWinRM.ps1"
+    winrm_insecure = true
+    winrm_timeout  = "20m"
+    winrm_use_ssl  = true
+    winrm_username = local.windows_winrm_user[var.image_type]
+  }
+
+  source "azure-arm.base" {
+    name            = "windows"
+    communicator    = "winrm"
+    image_offer     = "WindowsServer"
+    image_publisher = "MicrosoftWindowsServer"
+    image_sku       = "${var.agent_os_version}-datacenter-core-with-containers-smalldisk-g2"
+    vm_size         = local.azure_vm_size
+    os_type         = "Windows"
+    os_disk_size_gb = local.windows_disk_size_gb
+    winrm_insecure  = true
+    winrm_timeout   = "20m"
+    winrm_use_ssl   = true
+    winrm_username  = local.windows_winrm_user[var.image_type]
+  }
+
+  ## Why repeating? https:#github.com/rgl/packer-plugin-windows-update/issues/90#issuecomment-842569865
```

Comment:
What is preventing to put valid links in these comments?

Reply:
nothing, it was a typo. good catch!

Reply:
```suggestion
  ## Why repeating? https://github.com/rgl/packer-plugin-windows-update/issues/90#issuecomment-842569865
```